src/lib/components/__tests__/Details.svelte.spec.ts

```diff
@@ -76,13 +71,15 @@ describe('Details component', () => {
 	})
 
 	it('renders object name', () => {
-		render(Details)
+		const context = createWeblabs()
+		render(Details, { context: new Map([[WEBLABS_CONTEXT_KEY, context]]) })
```

Comment:
This is the idomatic way to test with contexts - and requires no mock, so the test is more reflective of real usage